moto/mediaconnect/responses.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import unicode_literals
+from moto.core.responses import BaseResponse
+from .models import mediaconnect_backends
+import json
+
+
+class MediaConnectResponse(BaseResponse):
+    SERVICE_NAME = "mediaconnect"
+
+    @property
+    def mediaconnect_backend(self):
+        return mediaconnect_backends[self.region]
+
+    def create_flow(self):
+        availability_zone = self._get_param("availabilityZone")
+        entitlements = self._get_param("entitlements")
+        name = self._get_param("name")
+        outputs = self._get_param("outputs")
+        source = self._get_param("source")
+        source_failover_config = self._get_param("sourceFailoverConfig")
+        sources = self._get_param("sources")
+        vpc_interfaces = self._get_param("vpcInterfaces")
+        flow = self.mediaconnect_backend.create_flow(
+            availability_zone=availability_zone,
+            entitlements=entitlements,
+            name=name,
+            outputs=outputs,
+            source=source,
+            source_failover_config=source_failover_config,
+            sources=sources,
+            vpc_interfaces=vpc_interfaces,
+        )
+        return json.dumps(dict(flow=flow.to_dict()))
+
+    def list_flows(self):
+        max_results = self._get_int_param("maxResults")
+        next_token = self._get_param("nextToken")
+        flows, next_token = self.mediaconnect_backend.list_flows(
+            max_results=max_results, next_token=next_token,
+        )
+        return json.dumps(dict(flows=flows, nextToken=next_token))
+
+    def describe_flow(self):
+        flow_arn = self._get_param("flowArn")
+        if flow_arn:
+            flow_arn = flow_arn.replace("%3A", ":")
```

Comment:
It looks like this can simply be replace with `unquote(arn)`
```
try:
    from urllib import unquote
except ImportError:
    from urllib.parse import unquote
```
That's how ARN's are decoded in other services.

Do we need the if-statement? It looks like the param should always be there, and if it isn't, we should fail anyway - so it might as well become `flow_arn = unquote(self._get_param("flowArn"))`

Reply:
You're right. I've made this change. Thank you!